### PR TITLE
Prepare for Github workflow

### PR DESCRIPTION
**Pull request by Halamix2**
**Description**

Changes proposed in this pull request:

- add CODEOWNERS file, updated with the current data

**Related issue(s)**

- https://github.com/kyma-project/test-infra/issues/5312
- https://github.com/kyma-project/test-infra/issues/5317

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+# Each line is a file pattern followed by one or more owners.
+# The last matching pattern has the most precedence.
+# For more details read the following article on GitHub: https://help.github.com/articles/about-codeowners/.
+
+# These are the default owners for the whole content of the `kyma` repository. The default owners are automatically added as reviewers when you open a pull request unless different owners are specified in the file.
+* @pbochynski @PK85 @a-thaler
+
+# All .md files
+*.md @klaudiagrz @mmitoraj @majakurcius @alexandra-simeonova @NHingerl @grego952
```
Comment (dekiel) [inline CODEOWNERS:11]: This line should be at the end to take precedence over all other matches.

**Review by dekiel — CHANGES_REQUESTED** [review]

File: CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+/tests/service-catalog @wozniakjan @PK85 @piotrmiskiewicz @szwedm @ralikio @hanngos @tillknuesting
+
+# The tools/event-subscriber directory
+/tools/event-subscriber/ @lilitgh @k15r @nachtmaar @marcobebway @radufa @pxsalehi @mfaizanse @friedrichwilken @vladislavpaskar @raypinto
+
+# The tools/gitserver directory
+/tools/gitserver/ @m00g3n @pPrecel @dbadura @kwiatekus @moelsayed @cortey
+
+# Config file for MILV - milv.config.yaml
+milv.config.yaml @m00g3n @pPrecel @kwiatekus @moelsayed @cortey @klaudiagrz @mmitoraj @majakurcius @alexandra-simeonova @NHingerl @grego952
```
Comment (majakurcius) [inline CODEOWNERS:130]:
```suggestion
milv.config.yaml @m00g3n @pPrecel @kwiatekus @moelsayed @cortey @klaudiagrz @mmitoraj @majakurcius @NHingerl @grego952
```

Comment (Halamix2) [reply CODEOWNERS]: Done

**Review by majakurcius — APPROVED** [review]

File: CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+/components/application-registry  @VOID404 @akgawlas @koala7659 @janmedrek @Maladie @franpog859 @mvshao
+/components/binding @wozniakjan @PK85 @piotrmiskiewicz @szwedm @ralikio @hanngos @tillknuesting
+/components/busola-migrator @m00g3n @pPrecel @dbadura @kwiatekus @moelsayed @cortey
+/components/central-application-connectivity-validator  @VOID404 @akgawlas @koala7659 @janmedrek @Maladie @franpog859 @mvshao
+/components/central-application-gateway  @VOID404 @akgawlas @koala7659 @janmedrek @Maladie @franpog859 @mvshao
+/components/compass-runtime-agent  @VOID404 @akgawlas @koala7659 @janmedrek @Maladie @franpog859 @mvshao @PK85 @dbadura
+/components/connection-token-handler  @VOID404 @akgawlas @koala7659 @janmedrek @Maladie @franpog859 @mvshao
+/components/connector-service  @VOID404 @akgawlas @koala7659 @janmedrek @Maladie @franpog859 @mvshao
+/components/event-publisher-proxy @k15r @nachtmaar @marcobebway @pxsalehi @mfaizanse @friedrichwilken @vladislavpaskar @raypinto @radufa @lilitgh
+/components/eventing-controller @k15r @nachtmaar @marcobebway @pxsalehi @mfaizanse @friedrichwilken @vladislavpaskar @raypinto @radufa @lilitgh
```
Comment (FriedrichWilken) [inline CODEOWNERS:36]:
```suggestion
/components/eventing-controller @k15r @nachtmaar @marcobebway @pxsalehi @mfaizanse @friedrichwilken @vladislavpaskar @raypinto @lilitgh
```

File: CODEOWNERS [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+# All files and subdirectories in /docs
+/docs @klaudiagrz @mmitoraj @majakurcius @NHingerl @grego952
+
+/hack @clebs
+
+/installation @khlifi411 @jeremyharisch @ruanxin @tobiscr @moelsayed @clebs @Tomasz-Smelcerz-SAP @jakobmoellersap @adityabhatia
+/installation/resources/crds/api-gateway @strekm @werdes72 @mjakobczyk @veichtj @dariusztutaj @cnvergence @piotrkpc
+/installation/resources/crds/application-connector  @VOID404 @akgawlas @koala7659 @janmedrek @Maladie @franpog859 @mvshao
+/installation/resources/crds/compass-runtime-agent  @VOID404 @akgawlas @koala7659 @janmedrek @Maladie @franpog859 @mvshao
+/installation/resources/crds/eventing @k15r @nachtmaar @marcobebway @pxsalehi @mfaizanse @friedrichwilken @vladislavpaskar @raypinto @radufa @lilitgh
```
Comment (FriedrichWilken) [inline CODEOWNERS:52]:
```suggestion
/installation/resources/crds/eventing @k15r @nachtmaar @marcobebway @pxsalehi @mfaizanse @friedrichwilken @vladislavpaskar @raypinto @lilitgh
```

File: CODEOWNERS [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+/resources/application-connector/charts/central-application-gateway  @VOID404 @akgawlas @koala7659 @janmedrek @Maladie @franpog859 @mvshao
+/resources/application-connector/charts/connection-token-handler  @VOID404 @akgawlas @koala7659 @janmedrek @Maladie @franpog859 @mvshao
+/resources/application-connector/charts/connector-service  @VOID404 @akgawlas @koala7659 @janmedrek @Maladie @franpog859 @mvshao
+/resources/busola-migrator @m00g3n @pPrecel @dbadura @kwiatekus @moelsayed @cortey
+/resources/certificates @mjakobczyk @veichtj @dariusztutaj @cnvergence @piotrkpc @strekm @werdes72
+/resources/cluster-essentials @janmedrek @a-thaler @strekm @dariusztutaj @cnvergence @veichtj @piotrkpc @PK85
+/resources/cluster-essentials/charts/pod-preset @wozniakjan @PK85 @piotrmiskiewicz @szwedm @ralikio @hanngos @tillknuesting
+/resources/cluster-users @cnvergence @piotrkpc @strekm @werdes72 @mjakobczyk @veichtj @dariusztutaj
+/resources/compass-runtime-agent  @VOID404 @akgawlas @koala7659 @janmedrek @Maladie @franpog859 @mvshao
+/resources/eventing @k15r @nachtmaar @marcobebway @pxsalehi @mfaizanse @friedrichwilken @vladislavpaskar @raypinto @radufa @lilitgh
```
Comment (FriedrichWilken) [inline CODEOWNERS:77]:
```suggestion
/resources/eventing @k15r @nachtmaar @marcobebway @pxsalehi @mfaizanse @friedrichwilken @vladislavpaskar @raypinto @lilitgh
```

**Review by FriedrichWilken — COMMENTED** [review]
Here are some proposed changes. There was a missing "@" and we need to remove @radufa from the code owner.

**Comment by koala7659** [comment]
Two comments:

1. I believe `framefrog` team should be mentioned as owners of fast integration tests directory. At least in the case of `/tests/fast-integration/skr-test` of the best full directory `/tests/fast-integration`.  As a team responsible for application connectivity we maintain and fix them regularly.
2.  There is no directory `/tests/fast-integration/skr-test-without-sc` on the main branch. Can you check if this entry is valid?

File: CODEOWNERS [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+/tests/application-gateway-tests  @VOID404 @akgawlas @koala7659 @janmedrek @Maladie @franpog859 @mvshao
+/tests/application-operator-tests  @VOID404 @akgawlas @koala7659 @janmedrek @Maladie @franpog859 @mvshao
+/tests/application-registry-tests  @VOID404 @akgawlas @koala7659 @janmedrek @Maladie @franpog859 @mvshao
+/tests/compass-runtime-agent  @VOID404 @akgawlas @koala7659 @janmedrek @Maladie @franpog859 @mvshao
+/tests/connection-token-handler-tests  @VOID404 @akgawlas @koala7659 @janmedrek @Maladie @franpog859 @mvshao
+/tests/connector-service-tests  @VOID404 @akgawlas @koala7659 @janmedrek @Maladie @franpog859 @mvshao
+
+# Fast Integration Tests
+/tests/fast-integration @lindnerby @skhalash @voigt @rakesh-garimella @jeremyharisch @khlifi411 @ruanxin @tobiscr @clebs @pbochynski @dennis-ge @jakobmoellersap @Tomasz-Smelcerz-SAP
+/tests/fast-integration/eventing-test @yma-project/eventing
```
Comment (FriedrichWilken) [inline CODEOWNERS:105]:
```suggestion
/tests/fast-integration/eventing-test @kyma-project/eventing
```

File: CODEOWNERS [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+/tests/application-gateway-tests  @VOID404 @akgawlas @koala7659 @janmedrek @Maladie @franpog859 @mvshao
+/tests/application-operator-tests  @VOID404 @akgawlas @koala7659 @janmedrek @Maladie @franpog859 @mvshao
+/tests/application-registry-tests  @VOID404 @akgawlas @koala7659 @janmedrek @Maladie @franpog859 @mvshao
+/tests/compass-runtime-agent  @VOID404 @akgawlas @koala7659 @janmedrek @Maladie @franpog859 @mvshao
+/tests/connection-token-handler-tests  @VOID404 @akgawlas @koala7659 @janmedrek @Maladie @franpog859 @mvshao
+/tests/connector-service-tests  @VOID404 @akgawlas @koala7659 @janmedrek @Maladie @franpog859 @mvshao
+
+# Fast Integration Tests
+/tests/fast-integration @lindnerby @skhalash @voigt @rakesh-garimella @jeremyharisch @khlifi411 @ruanxin @tobiscr @clebs @pbochynski @dennis-ge @jakobmoellersap @Tomasz-Smelcerz-SAP
+/tests/fast-integration/eventing-test @kyma-project/eventing
```
Comment (FriedrichWilken) [inline CODEOWNERS:105]:
```suggestion
@k15r @nachtmaar @marcobebway @pxsalehi @mfaizanse @friedrichwilken @vladislavpaskar @raypinto @lilitgh
```

File: CODEOWNERS [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+/tests/application-gateway-tests  @VOID404 @akgawlas @koala7659 @janmedrek @Maladie @franpog859 @mvshao
+/tests/application-operator-tests  @VOID404 @akgawlas @koala7659 @janmedrek @Maladie @franpog859 @mvshao
+/tests/application-registry-tests  @VOID404 @akgawlas @koala7659 @janmedrek @Maladie @franpog859 @mvshao
+/tests/compass-runtime-agent  @VOID404 @akgawlas @koala7659 @janmedrek @Maladie @franpog859 @mvshao
+/tests/connection-token-handler-tests  @VOID404 @akgawlas @koala7659 @janmedrek @Maladie @franpog859 @mvshao
+/tests/connector-service-tests  @VOID404 @akgawlas @koala7659 @janmedrek @Maladie @franpog859 @mvshao
+
+# Fast Integration Tests
+/tests/fast-integration @lindnerby @skhalash @voigt @rakesh-garimella @jeremyharisch @khlifi411 @ruanxin @tobiscr @clebs @pbochynski @dennis-ge @jakobmoellersap @Tomasz-Smelcerz-SAP
+@k15r @nachtmaar @marcobebway @pxsalehi @mfaizanse @friedrichwilken @vladislavpaskar @raypinto @lilitgh
```
Comment (FriedrichWilken) [inline CODEOWNERS:105]:
```suggestion

/tests/fast-integration/eventing-test @k15r @nachtmaar @marcobebway @pxsalehi @mfaizanse @friedrichwilken @vladislavpaskar @raypinto @lilitgh
```

Comment (FriedrichWilken) [reply CODEOWNERS:105]: sorry, my bad

File: CODEOWNERS [hunk 9]
```diff
@@ -0,0 +1,136 @@
...
+/resources/cluster-users @cnvergence @piotrkpc @strekm @werdes72 @mjakobczyk @veichtj @dariusztutaj
+/resources/compass-runtime-agent  @VOID404 @akgawlas @koala7659 @janmedrek @Maladie @franpog859 @mvshao
+/resources/eventing @k15r @nachtmaar @marcobebway @pxsalehi @mfaizanse @friedrichwilken @vladislavpaskar @raypinto @lilitgh
+/resources/helm-broker @wozniakjan @PK85 @piotrmiskiewicz @szwedm @ralikio @hanngos @tillknuesting
+/resources/istio @strekm @werdes72 @mjakobczyk @veichtj @dariusztutaj @cnvergence @piotrkpc
+/resources/istio-resources @strekm @werdes72 @mjakobczyk @veichtj @dariusztutaj @cnvergence @piotrkpc
+/resources/istio-resources/files/dashboards @a-thaler @clebs @rakesh-garimella @skhalash @jeremyharisch @chrkl @lindnerby @dennis-ge @lilitgh
+/resources/kiali @a-thaler @clebs @rakesh-garimella @skhalash @jeremyharisch @chrkl @lindnerby @dennis-ge @lilitgh
+/resources/logging @a-thaler @clebs @rakesh-garimella @skhalash @jeremyharisch @chrkl @lindnerby @dennis-ge @lilitgh
+/resources/monitoring @a-thaler @clebs @rakesh-garimella @skhalash @jeremyharisch @chrkl @lindnerby @dennis-ge @lilitgh
```
Comment (lilitgh) [inline CODEOWNERS:84]:
```suggestion
/resources/monitoring @a-thaler @rakesh-garimella @skhalash @chrkl @lindnerby @dennis-ge @lilitgh
```

File: CODEOWNERS [hunk 10]
```diff
@@ -0,0 +1,136 @@
...
+/resources/ory @strekm @werdes72 @mjakobczyk @veichtj @dariusztutaj @cnvergence @piotrkpc
+/resources/rafter @m00g3n @pPrecel @dbadura @kwiatekus @moelsayed @cortey
+/resources/serverless @m00g3n @pPrecel @dbadura @kwiatekus @moelsayed @cortey
+/resources/service-catalog @wozniakjan @PK85 @piotrmiskiewicz @szwedm @ralikio @hanngos @tillknuesting
+/resources/service-catalog-addons @wozniakjan @PK85 @piotrmiskiewicz @szwedm @ralikio @hanngos @tillknuesting
+/resources/service-manager-proxy @wozniakjan @PK85 @piotrmiskiewicz @szwedm @ralikio @hanngos @tillknuesting
+/resources/telemetry @a-thaler @clebs @rakesh-garimella @skhalash @jeremyharisch @chrkl @lindnerby @dennis-ge @lilitgh
```
Comment (lilitgh) [inline CODEOWNERS:91]:
```suggestion
/resources/telemetry @a-thaler @rakesh-garimella @skhalash @chrkl @lindnerby @dennis-ge @lilitgh
```

File: CODEOWNERS [hunk 11]
```diff
@@ -0,0 +1,136 @@
...
+/resources/tracing @a-thaler @clebs @rakesh-garimella @skhalash @jeremyharisch @chrkl @lindnerby @dennis-ge @lilitgh
```
Comment (lilitgh) [inline CODEOWNERS:92]:
```suggestion
/resources/tracing @a-thaler @rakesh-garimella @skhalash @chrkl @lindnerby @dennis-ge @lilitgh
```

**Pull request merged**